### PR TITLE
more unit tests + some fixes found during testing

### PR DESCRIPTION
**Pull request by peuter**

### CODE REVIEW

**Comment by ChristianMayer** [comment]
+7,3% to go up to 40.722% is great!

File: src/structure/pure/InfoTrigger.js [hunk 1]
```diff
@@ -70,10 +71,10 @@ define( ['_common'], function( design ) {
     actorup += '<div class="label">' + (data.uplabel || '+') + '</div>';
     actorup += '</div>';
 
-    var actorinfo = '<div class="actor switchInvisible " ';
+    var actorinfo = '<div class="actor switchInvisible" ';
     if ( data.align ) 
       actorinfo += 'style="text-align: ' + data.align + '" '; 
-    actorinfo += '" ><div class="value">-</div></div>';
```
Comment (ChristianMayer) [inline src/structure/pure/InfoTrigger.js]: good catch!

File: src/structure/pure/MultiTrigger.js [hunk 2]
```diff
@@ -51,54 +51,35 @@ define( ['_common'], function( design ) {
...
     ret_val += '<div class="actor_container" style="float:left">';
-    var buttonCount = 0;
-    
+
     if( data.button1label )
     {
-      ret_val += '<div class="actor switchUnpressed ';
-      if( data.align ) 
-        ret_val += data.align; 
-      ret_val += '">';
```
Comment (ChristianMayer) [inline src/structure/pure/MultiTrigger.js]: Why did you remove the align code?

Comment (peuter) [inline src/structure/pure/MultiTrigger.js]: Because there is no align attribute defined for multitriggers in the XSD, so this is unreachable code.
